src/test/java/core/basesyntax/RegistrationServiseTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiseTest {
+    private static StorageDao dao;
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    public static void getStorage() {
+        dao = new StorageDaoImpl();
```

Comment:
```suggestion
        dao = new StorageDaoImpl();
```
I don't have this, but all is working. I'm not sure if it is needed